tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml

```diff
@@ -0,0 +1,103 @@
+##############################################################
+#####  This file is dedicated for 202012 branch issues   #####
+#####  sonic-mgmt issue should be filed and linked here  #####
+##############################################################
+
+dualtor/test_ipinip.py::test_decap_standby_tor
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'dualtor' not in topo_name"
+
+dualtor/test_orchagent_active_tor_downstream.py
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'7260' in platform"
+
+dualtor/test_orchagent_mac_move.py
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'dualtor' not in topo_name"
+
+dualtor/test_orch_stress.py::test_flap_neighbor_entry_standby
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'dualtor' not in topo_name"
+
+dualtor/test_standby_tor_upstream_mux_toggle.py
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'dualtor' not in topo_name"
+
+dualtor/test_tor_ecn.py
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+
+dualtor_io/test_normal_op.py
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'dualtor' in topo_name"
+      - "'7260' in platform"
+
+platform_tests/api/test_chassis.py::TestChassisApi
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+
+platform_tests/test_advanced_reboot.py
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'dualtor' in topo_name"
```

Comment:
@StormLiangMS For this case, I found it failed on both dualtor and singal tor. Should we remove this dualtor condition?

Reply:
sure.